packages/components/src/test/instance-filter-builder/IntanceFilterBuilder.test.tsx

```diff
@@ -48,7 +49,21 @@ describe("InstanceFilter", () => {
     sinon.restore();
   });
 
+  const property: PropertyDescription = {
+    name: "propertyField1",
+    displayLabel: "Prop1",
+    typename: "boolean",
+  };
+
+  const propertyFilter: PropertyFilter = {
+    property,
+    operator: PropertyFilterRuleOperator.IsNull,
+    value: undefined,
+  };
+
   it("invokes 'onClassSelected' when non selected class is clicked", () => {
+    const { result } = renderHook(() => usePropertyFilterBuilder({ initialFilter: propertyFilter }));
```

Comment:
Can we mock `rootGroup` and `actions` in these tests?

Reply:
mocked